src/components/organisms/BalanceGameList/BalanceGameList.tsx

```
@@ -12,6 +12,7 @@ import { useNavigate } from 'react-router-dom';
 import { ERROR } from '@/constants/message';
 import { PATH } from '@/constants/path';
 import MobileToggleGroup from '@/components/mobile/atoms/MobileToggleGroup/MobileToggleGroup';
+import { PATH } from '@/constants/path';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'PATH'. Consider to delete it or rename it.

'PATH' is defined here:


(lint/suspicious/noRedeclare)